cognite/transformations_cli/commands/utils.py

```diff
@@ -4,20 +4,20 @@
 
 import click
 import sqlparse
-from cognite.experimental import CogniteClient as ExpCogniteClient
-from cognite.experimental.data_classes.transformation_jobs import TransformationJob, TransformationJobMetric
-from cognite.experimental.data_classes.transformation_notifications import TransformationNotification
-from cognite.experimental.data_classes.transformations import (
-    RawTable,
+from cognite.client import CogniteClient
+from cognite.client.data_classes import (
     Transformation,
     TransformationDestination,
-    TransformationPreviewResult,
+    TransformationJob,
+    TransformationJobMetric,
+    TransformationNotification,
 )
+from cognite.client.data_classes.transformations import RawTable, TransformationPreviewResult
```

Comment:
I think these can go back in the above import from `cognite.client.data_classes` with the last sdk version.